include/sparse_bundle_adjustment/csparse.h

```diff
@@ -52,14 +52,14 @@
 
 // CSparse header
 extern "C" {
-#include "suitesparse/cs.h"
+#include "cs.h"
```

Comment:
this is breaking the build for ubuntu. I think instead of changing the includes you really need to look at adding proper include paths in CMake for OS X.

Reply:
Agreed - I can figure that out later once it builds correctly on OSX.